### ISSUE TITLE
Parameters in SELECT queries do not support the timezone in date, datetime

### ISSUE THREAD

**Issue by sc07kvm** [mymarilyn/clickhouse-driver#268]
`datetime` and `date` in the select query are escaped regardless of the timezone.

```
t = datetime.datetime(2021, 11, 10, 12, 30, 15, tzinfo=datetime.timezone(datetime.timedelta(seconds=10800)))
client.execute('select * from table1 where table.datetime < %(datetime)s', {'datetime': t})
```
At this time in the server logs(`select query from system.query_log)`:
```
select * from table1 where table.datetime < '2021-11-10 12:30:15
```
**Expected behavior**
```
select * from table1 where table.datetime < '2021-11-10 9:30:15'
```

**Versions**
- clickhouse-driver 0.2.2
- ClickHouse 21.6.4.26
- Python 3.9

**Comment by xzkostyan** [comment]
Plain date should be escaped regardless of the timezone.

If datetime parameter is timezone aware, it will be converted into server's timezone: https://github.com/mymarilyn/clickhouse-driver/commit/8b8bdb6fe2b46d93f1107a34e3dc306d85fe7dfd

**Comment by xzkostyan** [comment]
Fixed in 0.2.3.
